packages/service-clients/end-to-end-tests/azure-client/src/test/AzureClientFactory.ts

```diff
@@ -70,3 +71,60 @@ export function createAzureClient(
 		configProvider,
 	});
 }
+
+export async function createContainerFromPayload(
+	requestPayload: object,
+	userID?: string,
+	userName?: string,
+): Promise<AxiosResponse | undefined> {
+	const useAzure = process.env.FLUID_CLIENT === "azure";
+	const tenantId = useAzure
+		? (process.env.azure__fluid__relay__service__tenantId as string)
+		: "frs-client-tenant";
+	const user = {
+		id: userID ?? uuid(),
+		name: userName ?? uuid(),
+	};
+	const endPoint = useAzure
+		? (process.env.azure__fluid__relay__service__endpoint as string)
+		: "http://localhost:7071";
+	if (useAzure && endPoint === undefined) {
+		throw new Error("Azure FRS endpoint is missing");
+	}
+
+	const tokenProvider = useAzure
+		? createAzureTokenProvider(userID ?? "foo", userName ?? "bar")
+		: new InsecureTokenProvider("fooBar", user);
+	const ordererToken = await tokenProvider.fetchOrdererToken(tenantId, undefined, false);
+
+	const headers = {
+		"Authorization": `Basic ${ordererToken.jwt}`,
+		"Content-Type": "application/json",
+	};
+
+	const url = `/documents/${tenantId}`;
+
+	const options: AxiosRequestConfig = {
+		baseURL: endPoint,
+		data: requestPayload,
+		headers,
+		maxBodyLength: 1048576000,
+		maxContentLength: 1048576000,
+		method: "POST",
+		url,
+	};
+
+	try {
+		const response: AxiosResponse = await Axios(options);
+
+		if (response.status === 201) {
+			console.log("Container created successfully");
+		} else {
+			console.error(`Error creating container. Status code: ${response.status}`);
+		}
+		return response;
+	} catch (error) {
+		console.error("An error occurred:", error);
```

Comment:
Why suppress the error?  Shouldn't the test fail immediately if we hit errors or unexpected response codes?

Reply:
Changed from logging the error to throwing it